docs/development/index.rst

```diff
@@ -15,7 +15,7 @@ check out `"What to put in your bug report"`_ for guidance.
                details.
 
 You can also join ``#pypa`` (general packaging discussion and user support) and
-``#pypa-dev`` (discussion about development of packaging tools) `on Freenode`_,
+``#pypa-dev`` (discussion about development of packaging tools) `on Freenode`_, `Libera Chat`_, `Discord`_,
```

Comment:
- I think we should remove Freenode entirely
- `#pypa` and `#pypa-dev` make sense for Libera but not for Discord. Maybe the sentence should be turned around ?

Reply:
So Freenode won't be used anymore?

and also should I add one more sentence for Discord after `to ask questions or get involved.` ?

Reply:
Freenode is basically dead. They have deleted all channels and usernames and invited people to recreate everything from scratch, but I don't think anyone considers it a serious network anymore.

Also, the distutil-sig is hardly the place where discussion happens anymore, it's been mostly replaced by the packaging category in discuss.python.org .

I'd suggest:
```
Get involved using:
- IRC: on Libera_, channel ``#pypa`` for general packaging discussion and user support, and
  ``#pypa-dev`` for discussions about development of packaging tools
- The `PyPA Discord`_ for live discussions
- The Packaging category on `Discourse` for discussing new ideas and community initiatives

```

Reply:
Thanks

May you also provide the link of the Packaging category on Discourse, please?

Reply:
> May you also provide the link of the Packaging category on Discourse, please?

It's in my message just above :D
In case it's unclear, "Discuss" == "Discourse" ("discuss.python.org" is the URL of the Discourse instance for the Python community)

https://discuss.python.org/c/packaging/14

Reply:
Oh really? 🤣

Reply:
now I will change to old one to [this](https://github.com/pypa/warehouse/pull/9690#discussion_r653429420)